library/openjdk

```diff
@@ -1,53 +1,53 @@
-# this file is generated via https://github.com/docker-library/openjdk/blob/91d42d242a68fa0b8a4cd598c9fac670d4807544/generate-stackbrew-library.sh
+# this file is generated via https://github.com/docker-library/openjdk/blob/3eb0351b208d739fac35345c85e3c6237c2114ec/generate-stackbrew-library.sh
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
 GitRepo: https://github.com/docker-library/openjdk.git
 
-Tags: 13-ea-21-jdk-oraclelinux7, 13-ea-21-oraclelinux7, 13-ea-jdk-oraclelinux7, 13-ea-oraclelinux7, 13-jdk-oraclelinux7, 13-oraclelinux7, 13-ea-21-jdk-oracle, 13-ea-21-oracle, 13-ea-jdk-oracle, 13-ea-oracle, 13-jdk-oracle, 13-oracle
-SharedTags: 13-ea-21-jdk, 13-ea-21, 13-ea-jdk, 13-ea, 13-jdk, 13
+Tags: 13-ea-22-jdk-oraclelinux7, 13-ea-22-oraclelinux7, 13-ea-jdk-oraclelinux7, 13-ea-oraclelinux7, 13-jdk-oraclelinux7, 13-oraclelinux7, 13-ea-22-jdk-oracle, 13-ea-22-oracle, 13-ea-jdk-oracle, 13-ea-oracle, 13-jdk-oracle, 13-oracle
+SharedTags: 13-ea-22-jdk, 13-ea-22, 13-ea-jdk, 13-ea, 13-jdk, 13
 Architectures: amd64
-GitCommit: ac186a1549186876b0d05babd4d909f209f27d59
+GitCommit: 448901d68e2a32f49bda1a8789d4c56e821506aa
 Directory: 13/jdk/oracle
 
 Tags: 13-ea-19-jdk-alpine3.9, 13-ea-19-alpine3.9, 13-ea-jdk-alpine3.9, 13-ea-alpine3.9, 13-jdk-alpine3.9, 13-alpine3.9, 13-ea-19-jdk-alpine, 13-ea-19-alpine, 13-ea-jdk-alpine, 13-ea-alpine, 13-jdk-alpine, 13-alpine
```

Comment:
Was this alpine variant left intentionally?

Reply:
Yeah, that one comes from https://jdk.java.net/13/, and the caveats on that page apply (it's essentially a preview build of the Portola work, as I've understood it).

12 and 11 had similar builds that also went away when they became GA.